src/errors.ts

```diff
@@ -71,33 +270,50 @@ export class KycDaoSDKError extends Error {
  * @internal
  * @param {Error} error
```

Comment:
```suggestion
 * @param {unknown} error
```
Just to keep things updated.